Src/App.Release.config

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!-- Release version of App.config -->
 <configuration>
-  <configSections>
-    <section name="microsoft.scripting" type="Microsoft.Scripting.Hosting.Configuration.Section, Microsoft.Scripting, Version=1.1.2.22, Culture=neutral, PublicKeyToken=7f709c5b713576e1" requirePermission="false"/>
```

Comment:
Any reason this is completely removed for release but included in debug?

Reply:
I forgot to remove it from the debug version.